src/accounts/AccountPage.tsx

```diff
@@ -34,9 +33,6 @@ import AccountTabContainer from 'src/accounts/AccountTabContainer'
 import AccountHeader from 'src/accounts/AccountHeader'
 import {DeleteFreeAccountProvider} from 'src/accounts/context/DeleteFreeAccountContext'
 
-import CancellationOverlay from './CancellationOverlay'
-import CancelServiceProvider from 'src/billing/components/PayAsYouGo/CancelServiceContext'
```

Comment:
This is called CancellationOverlay, but there are actually a couple overlapping 'Cancellation' overlays in the code, one of which is for PAYG. This one would better be called FreeAccountCancellationOverlay, but we're deleting it.

We're safe to remove this particular overlay, because it's only used in the context of free account cancellation from what I can tell (I tested it by eliminating the default export and seeing what relied on that).